extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KnativeProcessor.java

```diff
@@ -192,6 +193,7 @@ public List<DecoratorBuildItem> createDecorators(ApplicationInfoBuildItem applic
         config.maxScale.map(String::valueOf).ifPresent(max -> result.add(new DecoratorBuildItem(KNATIVE,
                 new ApplyAnnotationsToServiceTemplate(name, KNATIVE_MAX_SCALE, max))));
         /**
+         * Βσ
```

Comment:
?

Reply:
LOL. Who doesn't want random greek characters?

Reply:
Its all greek to me :D